app/controllers/internal/feedback_messages_controller.rb

```diff
@@ -2,23 +2,28 @@ class Internal::FeedbackMessagesController < Internal::ApplicationController
   layout "internal"
 
   def index
-    @q = FeedbackMessage.
-      includes(:reporter,
-               :offender,
-               :affected).
-      order("feedback_messages.created_at DESC").
-      ransack(params[:q])
-    @feedback_messages = @q.result.page(params[:page] || 1).per(5)
+    @view = params[:view]
+    if @view == "new-articles"
```

Comment:
if we split these `view == ` in different routes we gain an easier code organization, so instead of `/internals/reports?view=name` it could be `/internal/reports/name` and each of the view has its own action method